libsession/src/main/java/org/session/libsession/avatars/PlaceholderAvatarPhoto.kt

```diff
@@ -1,13 +1,10 @@
 package org.session.libsession.avatars
 
-import android.content.Context
 import com.bumptech.glide.load.Key
 import java.security.MessageDigest
 
-class PlaceholderAvatarPhoto(val context: Context,
```

Comment:
`Key` is a used to cache an image for Glide, `context` should not be part of an image Key (it doesn't make sense anyway), and the cache key is stored in a global object.